src/data/taskList.ts

```diff
@@ -46,13 +46,25 @@ export const bizCaseStatus = (
   businessCase: BusinessCaseModel
   // eslint-disable-next-line consistent-return
 ) => {
+  // This is for our NEW and CURRENT intake statuses
+  // until all the old ones are deprecated
+  switch (intakeStatus) {
+    case 'BIZ_CASE_DRAFT_SUBMITTED':
+      return 'BIZ_CASE_DRAFT_SUBMITTED';
+    default:
+      break;
+  }
+
+  // START: Delete this when old statuses are deprecated
   if (businessCase === businessCaseInitialData) {
```

Comment:
If the status isn't found in the `switch` above, that means the status isn't a part of the NEW intake statuses that we defined. So we just go through the old flow.

As the comment indicates, all this should get deleted once our new statuses are hooked up to the actions API.